Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -1,15 +1,4 @@
-﻿// ---------------------------------------------------------------------------------------
```

Comment:
MINOR: Please add the copyright header back

Reply:
I don't  remember removing it, that's odd. It must have been an accident. Can I just copy paste it from any other file?

Reply:
Also, should the copyright header be included in the text template?

Reply:
Anyways, it appears to be that way in MemoryBuffer.tt, so I'll follow the convention.